src/utils.js

```diff
@@ -32,18 +32,46 @@ exports.getRoutes = routerStack => {
       });
     }
 
+    /**
+     *  checking to see if the word api exists, if so,
+     *  it is being removed separetely from the string and 
+     *  also the v1 or v2 is being removed separately.
+     *  Everything is then concatenated into the 'fullUrlPath'
+     *  variable.
+     */    
+
     if (stacks.name === 'router') {
       const { handle } = stacks;
-      const version = stacks
+
+      const strigifiedRegex = stacks
         .regexp
-        .toString()
-        .replace(/[fast_car:false\\?i=|$)()/^true]/ig, '');
+        .toString();      
+
+      const baseurl = strigifiedRegex.search(/api/);
```

Comment:
this has the assumption that the base url will always start with `api` but it could be anything

Reply:
Anything like, cos I feel most Base URL always starts with api. I'd like to know what other names the base url could be called.

Reply:
And then agian, did you delete your twitter account?

Reply:
Yeah, so we have to programmatically get it out. We can't always assume it's `api`

I deactivated my twitter account temporarily.

Reply:
Alright. I'll look into it.

Reply:
I just made a change to the PR